action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
     description: 'If true, a comment will be created with results'
     required: false
     default: 'false'
+  comment-on-pr-number:
+    description: 'Overrides the PR used to comment on'
+    required: false
+    default: '0'
```

Comment:
Do we need to set a default?

Reply:
```suggestion
```

I think we probably could leave it out, but I can do that later (I'm going to refactor some of our config setup)